mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,144 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, win_length, n_harmonics):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    win_length : float
+        Length of window to use for SNR estimates (seconds). A longer window
+        will naturally include more low frequency power, resulting in lower
+        SNR.
+    n_harmonics : int
```

Comment:
What would be the most reasonable default for "auto"? Something ilke all harmonics up to 400 Hz (or lower depending on `info['lowpass'`] and `info['sfreq']`? That would probably get most of the bad ones in most of the cases, right?